Finmer.Game/Models/UserConfig.cs

```diff
@@ -92,6 +97,7 @@ public static void Reload()
                         // Extract properties we're interested in
                         Hyphenation = props.GetBool(@"hyph");
                         PreferScat = props.GetBool(@"scat");
+                        PreySense = props.GetBool(@"prey");
```

Comment:
- Don't forget to save it as well! You're missing a matching line in `Flush()`.
- Just use "preysense" as a key, "prey" isn't really descriptive.